app.js

```diff
@@ -0,0 +1,77 @@
+
+console.log("Hello, World!"); //->English
+console.log("Silav, Cihan!");//->Kurdish
+console.log("Hoi, Wereld!"); //->Dutch
+
+console.log("I'm awesome!");
+
+let x =
+console.log('I think the value of x is bigger than 5');
+console.log("The value of x is 6");
+let x = 6;
+console.log(I think the value of x is 6);
+console.log(x);
+
+let y ="Sivan";
+console.log("I think the value of y is a name");
+console.log(y);
+const y = "Hassam";
+console.log("There will be an error because now I have
+             the same variable with different values.");
+console.log(y);
+
+const z = 7.25;
+console.log(z);
+const a = z;
+console.log(a);
+var round =Math.round(7.25);
+console.log(round)
+
+let arrays =
+console.log("I think variable arrays is related to animals");
+console.log(arrays);
+let myFavoriteAnimals = [cat, dog, sheep];
+console.log(myFavoriteAnimals);
+let favoriteAnimalOfDaan = "baby pig";
+const animals = "myFavoriteAnimals" + "" + "favoriteAnimalOfDaan";
+console.log(animals);
+
+let myString = "this is a test";
+console.log(myString);
+console.log(myString ["length"]);
+
+let num = 11;
+let address = "Huygenhoekring";
+let supermarkets = ["Lidl", "Dekka", "Jumbo"];
+let booleans = true, false, true;
+console.log(num);
+console.log(address);
+console.log(supermarkets);
+console.log(booleans);
+console.log(I think the type of num is number,

```

Comment:
Did you forget tot add " " in the console log? (because I dont think this compiles)

Reply:
Changed to: console.log("I think the type of num is number, type of address is string, type of supermarkets is array and type of booleans is boolean.");